homeassistant/components/logbook.py

```diff
@@ -189,9 +189,6 @@ def humanify(events):
             if event.event_type == EVENT_STATE_CHANGED:
                 entity_id = event.data.get('entity_id')
 
-                if entity_id is None:
-                    continue
-
                 if entity_id.startswith(tuple('{}.'.format(
```

Comment:
Creating a tuple for each event is also slow. We should move this out of the for loop.